lib/internal/url.js

```diff
@@ -263,6 +256,48 @@ class URL {
     ret += '}';
     return ret;
   }
+
+  [kFormat](options) {
```

Comment:
Symbols added this way are enumerable(can be verified via `Object.getOwnPropertyDescriptor(url, kFormat)`), I believe `test-whatwg-url-properties` doesn't catch this because `for-in` only iterates over string keys, but since `Reflect.enumerate` is withdrawn in ES2016 I can't think of a way to make that test more complete...anyways I think we should use`Object.defineProperty` here to make it non-enumerable, or just move this function out and use `.call` to call it properly?